main.tf

```diff
@@ -104,10 +108,10 @@ module "graphdb" {
 
   # VMs
 
-  instance_type   = var.instance_type
-  node_count      = var.node_count
-  userdata_script = module.graphdb.graphdb_userdata_base64_encoded
-  key_name        = var.key_name
+  ec2_instance_type   = var.ec2_instance_type
+  node_count          = var.node_count
```

Comment:
Now that the rest have some kind of a prefix, `node_count` needs one too. Could be `graphdb_node_count`